app/components/memorable_date_component.html.erb

```diff
@@ -1,9 +1,9 @@
 <label class="usa-label">     
     <%= label %>
 </label>
-<span class="usa-hint">
+<div class="usa-hint">
```

Comment:
The standout thing for me is that there is no relationship defined between the fields and this hint, unlike the default ValidatedFieldComponent, which assigns an ID to the hint and associates the field with it using `aria-describedby`. Should we be doing that here too?

https://github.com/18F/identity-idp/blob/592623ad6aa9ea4ffc2992fdc98171fbc3067704/app/components/validated_field_component.html.erb#L19-L30

Reply:
that's an interesting point. It might indeed make sense to add an aria describedby label to the hint in order to associate it with the field it's coupled with by the field's id. The way we're doing it in those other examples though is currently raising a red flag with one of the accessibility checkers I used. I still have to dig into what about the id is causing a violation (perhaps b/c the same id is appended onto the describedby property in multiple places and maybe it should instead have the id of the field? not 100% sure)

Reply:
To me, this would seem to explain why the hint isn't being announced, since the screen reader wouldn't have any indication that the text has anything to do with the field when the field is focused.

Reply:
i see what you're saying. So question I have now is this - adding the aria-describedby reads out the example for the memorable date component when the user navigates to the month, day and year input fields. This is what we'd expect, buuuuut- does it make sense to a user to hear "example 4 28 1986" for the month, then the day and then the year? because really for month an example would be just "4". Or maybe I'm overthinking it. I'm gonna put this in slack also so folks can chime in.

Reply:
Maybe this is a good use case for [`aria-placeholder`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-placeholder) instead.

Reply:
> i see what you're saying. So question I have now is this - adding the aria-describedby reads out the example for the memorable date component when the user navigates to the month, day and year input fields. This is what we'd expect, buuuuut- does it make sense to a user to hear "example 4 28 1986" for the month, then the day and then the year? because really for month an example would be just "4". Or maybe I'm overthinking it. I'm gonna put this in slack also so folks can chime in.

That's a good question! I agree with your concern that it would be confusing to say the full example text given how we've split the input values across three fields.

To this and @NavaTim 's suggestion, I'd also want to be careful about introducing inconsistency between the visual text and how it's conveyed to assistive technology (similar but not same as [G211](https://www.w3.org/WAI/WCAG21/Techniques/general/G211.html)). This _might_ be an acceptable case to make an exception, however.

For what it's worth, the [canonical markup example from the design system](https://designsystem.digital.gov/components/memorable-date/) uses `aria-describedby` with the same (full) hint example for each input.

Reply:
> For what it's worth, the [canonical markup example from the design system](https://designsystem.digital.gov/components/memorable-date/) uses aria-describedby with the same (full) hint example for each input, and typically we should be able to trust that they've put some thought into the accessibility implications.

That's a good point. Since they included the full example as a hint, I would be inclined to do the same.

Reply:
yes i agree -it makes sense for us to remain consistent with how they've implemented the hint. i'll go ahead and push up the change.